content/en/tracing/llm_observability/tracing_llm_applications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Tracing LLM Applications
+kind: guide
+---
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+## Overview
+
+Your application can submit data to LLM Observability in two ways: with Datadog's [Python SDK][12], or with the [LLM Observability API][13].
+
+Each request fulfilled by your application is represented as a trace on the [LLM Observability traces list page][2] in Datadog:
+
+{{< img src="tracing/llm_observability/llm-observability-overview.png" alt="An LLM Observability trace displaying each span of a request" style="width:90%;" >}}
+
+A given trace contains spans representing each choice made by an agent or each step of a workflow. A *span* represents some unit of work that your application is performing. Spans have a start time, duration, name, tags, and attributes.
+
+Multiple spans combine to form a trace, and a *root span* is the first span in a trace.
+
+A trace can contain several kinds of spans: Agent, LLM, Workflow, and so on. The *span kind* categorizes the type of work the span is performing. Different span kinds have different parent-child relationships, and only some kinds can be the root span of a trace. For details, see [Span Kinds][4].
+
+## Instrument an LLM application
+
+<div class="alert alert-info">This guide uses the Python SDK. If your application is not written in Python, you can complete the steps below with API requests instead of SDK function calls.</a></div>
+
+To trace a given LLM application:
+
+1. [Install the Python SDK][1].
+1. [Start your application with the required env variables][5], including your Datadog API key. If you don't have an API key, you can [create one in Datadog][3].
+1. In your code, [use the SDK to create spans](#create-spans) representing your application's tasks.
+    - See the span creation example below.
+    - For additional examples and detailed usage, see the [Quickstart][10] and the [SDK documentation for tracing spans][11]. 
+1. [Annotate your spans][7] with input data, output data, metadata (such as `max_tokens`), and key-value tags (such as `version:1.0.0`).
+1. Explore the resulting traces on the [traces list page][2].
+
+Optionally, you can also:
+
+- Associate multiple interactions with one user by specifying a `session_id`. See [Tracking user sessions][6] in the SDK documentation.
+- [Persist a span between contexts or scopes][8] by manually starting and stopping it.
+- [Override the name of the LLM application with a different name][9] when starting a root span, which can be useful for differentiating between services or running an experiment.
+
```

Comment:
```suggestion
- If you are collecting any feedback from your end users in your LLM application (e.g. thumbs up/thumbs down, rating experience out of 1 to 5, etc), you can submit [custom evaluation metrics via the API](/llm_observability/api/?tab=model#eval-metrics-api) so that you can visualize these feedback in your traces and also in the cluster view, where you can monitor any patterns in different topics of your LLM applications against these metrics.
```

Reply:
could we make sure this is included? thanks :)